demos/http3/ossl-nghttp3-demo-server.c

```diff
@@ -792,7 +792,7 @@ static int get_file_length(struct h3ssl *h3ssl)
 static char *get_file_data(struct h3ssl *h3ssl)
 {
     char filename[PATH_MAX];
-    int size = get_file_length(h3ssl);
+    size_t size = get_file_length(h3ssl);
```

Comment:
Hmm... there are more problems with the variable type in get_file_length() - so this is converted from off_t to int to size_t.

Reply:
I think thats ok here now, as off_t is (strangely) defined as a signed 32 bit type (even on 64 bit platforms), so as long as the return value is placed into a size_t on return, it should work out

Reply:
This should be OK for now, but I'm with t8m the cast from off_t to int which happens in get_file_length() is 64-bit unfriendly. off_t is usually int64_t.